dashboard/modules/job/job_manager.py

```diff
@@ -459,7 +459,10 @@ async def run(
                             "force-killed with SIGKILL."
                         )
                         self._kill_processes(proc_to_kill, signal.SIGKILL)
-                await self._job_info_client.put_status(self._job_id, JobStatus.STOPPED)
+                return_code = job_process.wait()
```

Comment:
This is blocking so we cannot use it directly in async.

I think we should use the same `_polling` method as the else case.